cmd/send.go

```
@@ -128,7 +128,7 @@ func init() {
 	sendCmd.Flags().BoolVarP(&forceUpdate, "force-update", "f", false, "Set flag to force update data sent to SMD")
 	sendCmd.Flags().StringVar(&cacertPath, "cacert", "", "Set the path to CA cert file (defaults to system CAs when blank)")
 
-	sendCmd.RegisterFlagCompletionFunc("format", completionFormatData)
+	_ = sendCmd.RegisterFlagCompletionFunc("format", completionFormatData)
```

See comment above regarding ignoring errors.